### PR TITLE
Merged Jervell-Lange Nielsen syndrome terms

### LINKED ISSUE #3073: review 'Jervell and Lange-Nielsen syndrome'
'Jervell and Lange-Nielsen syndrome' is a child of 'Jervell-Lange Nielsen syndrome' - should these be merged, or how did we get here?

Need to review.

### PR DESCRIPTION
**Pull request by sabrinatoro**
Fix #3073

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -38496,27 +38496,42 @@ is_a: MONDO:0020115 {source="MONDO:Redundant", source="NCIT:C35434"} ! secondary
...
 subset: gard_rare
+subset: ordo_clinical_subtype {source="Orphanet:90647"}
+subset: prototype_pattern
 synonym: "Cardioauditory syndrome of Jervell and Lange-Nielsen" RELATED [GARD:0003048]
 synonym: "deafness, congenital, and functional heart disease" RELATED [GARD:0003048]
+synonym: "Jervell and Lange Nielsen syndrome" EXACT [NCIT:C84793]
+synonym: "Jervell and Lange-Nielsen syndrome 1" RELATED [MONDO:Lexical, OMIM:220400]
+synonym: "Jervell and Lange-Nielsen syndrome type 1" RELATED [MONDORULE:1, OMIM:220400]
 synonym: "Jervell and Lange-Nielson syndrome" EXACT [DOID:2842]
 synonym: "Jervell Lange-Nielsen syndrome" RELATED [GARD:0003048]
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:38510]: could you make this exact? Chris automatically brought in a lot of synonyms and labeled them as related, when they really should be exact. thanks!

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -38496,27 +38496,42 @@ is_a: MONDO:0020115 {source="MONDO:Redundant", source="NCIT:C35434"} ! secondary
...
 xref: GARD:0003048 {source="MONDO:equivalentTo"}
+xref: ICD10:I45.8 {source="ORDO:90647/attributed", source="ORDO:90647/ntbt", source="Orphanet:90647"}
+xref: MedDRA:10057936 {source="ORDO:90647/e", source="Orphanet:90647"}
 xref: MESH:D029593 {source="DOID:2842", source="MONDO:equivalentTo"}
-xref: NCIT:C84793 {source="DOID:2842", source="MONDO:superClassOf"}
-xref: OMIM:220400 {source="GARD:0003048", source="DOID:2842", source="MONDO:superClassOf"}
-xref: OMIM:612347 {source="GARD:0003048", source="DOID:2842", source="MONDO:superClassOf"}
-xref: Orphanet:90647 {source="GARD:0003048", source="MONDO:superClassOf"}
-xref: SCTID:373905003 {source="DOID:2842", source="MONDO:superClassOf"}
+xref: NCIT:C84793 {source="MONDO:equivalentTo"}
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:38521]: we don't want to lose the source from DOID:2842. I manually merge these. Can you add DOID:2842 here please, thanks!

File: src/ontology/mondo-edit.obo [hunk 3]
```diff
@@ -38496,27 +38496,42 @@ is_a: MONDO:0020115 {source="MONDO:Redundant", source="NCIT:C35434"} ! secondary
...
+xref: OMIM:220400 {source="GARD:0003048", source="DOID:2842", source="MONDO:superClassOf", http://purl.obolibrary.org/obo/mondo#source="Orphanet:90647"}
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:38522]: not sure what is going on with these weird source annotations, this should be source"Orphanet:90647". Can you use the other source annotation in Protege? I thought Nico fixed this, but maybe it's still cropping up?

File: src/ontology/mondo-edit.obo [hunk 4]
```diff
@@ -38496,27 +38496,42 @@ is_a: MONDO:0020115 {source="MONDO:Redundant", source="NCIT:C35434"} ! secondary
...
+xref: OMIM:612347 {source="GARD:0003048", source="DOID:2842", source="MONDO:superClassOf", http://purl.obolibrary.org/obo/mondo#source="Orphanet:90647"}
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:38523]: same source issue as above

File: src/ontology/mondo-edit.obo [hunk 5]
```diff
@@ -38496,27 +38496,42 @@ is_a: MONDO:0020115 {source="MONDO:Redundant", source="NCIT:C35434"} ! secondary
...
+xref: OMIMPS:220400 {source="MONDO:cjm", source="MONDO:equivalentTo"}
+xref: Orphanet:768 {source="MONDO:subClassOf", source="OMIM:220400"}
+xref: Orphanet:90647 {http://purl.obolibrary.org/obo/mondo#source="GARD:0003048", source="MONDO:equivalentTo", source="OMIM:220400"}
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:38526]: same source issue as above

when you hover over the annotations in Protege, you can see where they come from.

image

File: src/ontology/mondo-edit.obo [hunk 6]
```diff
@@ -38496,27 +38496,42 @@ is_a: MONDO:0020115 {source="MONDO:Redundant", source="NCIT:C35434"} ! secondary
...
+xref: SCTID:373905003 {source="MONDO:equivalentTo"}
 xref: SCTID:49518001 {source="DOID:2842"}
-xref: UMLS:C0022387 {source="GARD:0003048", source="DOID:2842", source="MONDO:superClassOf"}
-is_a: MONDO:0002442 {source="DOID:2842", source="MESH:D029593"} ! long QT syndrome
+xref: UMLS:C0022387 {http://purl.obolibrary.org/obo/mondo#source="DOID:2842", http://purl.obolibrary.org/obo/mondo#source="GARD:0003048", source="MONDO:equivalentTo", source="NCIT:C84793", source="OMIM:220400", source="ORDO:90647/e", source="Orphanet:90647"}
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:38529]: same source issue as above

File: src/ontology/mondo-edit.obo [hunk 7]
```diff
@@ -38496,27 +38496,42 @@ is_a: MONDO:0020115 {source="MONDO:Redundant", source="NCIT:C35434"} ! secondary
...
+is_a: MONDO:0002442 {source="DOID:2842", source="MESH:D029593", http://purl.obolibrary.org/obo/mondo#source="NCIT:C84793"} ! long QT syndrome
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:38530]: same source issue as above

**Review by nicolevasilevsky — CHANGES_REQUESTED** [review]
a few small things, looks good otherwise, great job!

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:38510]: @nicolevasilevsky I don't think these should be exact synonyms. 
These refer to the type 1 version of the disease. This term refers to the grouping term (for both type 1 and type 2)

File: src/ontology/mondo-edit.obo (hunk 2 above)
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:38521]: done

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:38510]: You are right! I misread it. Thanks Sabrina :)

**Review by nicolevasilevsky — APPROVED** [review]

**Pull request merged**